common/djangoapps/util/milestones_helpers.py

```diff
@@ -32,7 +32,9 @@
 #   the Open edX platform. (see https://github.com/edx/edx-milestones)
 # .. toggle_use_cases: open_edx
 # .. toggle_creation_date: 2014-11-21
-# TODO this should be moved to edx/edx-milestones
+# .. toggle_target_removal_date: None
+# .. toggle_warnings: TODO this should be moved to edx/edx-milestones
```

Comment:
This won’t help operators, and it is debatable whether it will help owner any more than it being in code. Consider leaving this out of the annotation.